tableauserverclient/server/request_options.py

```diff
@@ -62,24 +62,49 @@ def apply_query_params(self, url):
         return "{0}?{1}".format(url, '&'.join(params))
 
 
-class ImageRequestOptions(RequestOptionsBase):
+class _FilterOptionsBase(RequestOptionsBase):
+    def __init__(self):
+        self.view_filters = []
+
+    def apply_query_params(self, url):
+        raise NotImplementedError()
+
+    def vf(self, name, value):
```

Comment:
There's no code testing this or sampling this yet, but I'm guessing this makes it:

```
opts = ImageRequestOptions()
opts.vf('a', 1).vf('b', 2) etc?
```

Baby-builder pattern! 
Did you still want to refactor this all to eventually just be builder-y and call `str` on those? (referenced in #212)

Reply:
Yes, eventually, we will refactor to a builder pattern!